integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/iud/DeleteCarbonTableTestCase.scala

```diff
@@ -322,7 +321,32 @@ class DeleteCarbonTableTestCase extends QueryTest with BeforeAndAfterAll {
 
   }
 
-  test("block deleting records from table which has index") {
+  // block delete operation from table which has index. see PR2483
+  test("delete should fail if the table has index") {
```

Comment:
Duplicate Test.
Please have a check about another testcase 'delete operation is not supported for index'

Reply:
ok